docs/manifest.json

```diff
@@ -108,6 +109,8 @@
             "name": "google/cloud-bigquery",
             "defaultService": "bigquery/bigqueryclient",
             "versions": [
+                "v1.2.4",
+                "v1.3.0",
```

Comment:
It looks like maybe a minor release was prepped and reverted, but not removed from this manifest.